External/SPMmodified/xASL/xASL_io_SaveNifti.m

```diff
@@ -234,4 +263,25 @@ function xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew, nBits, bGZip, cha
 end
 
 
+%% ====================================================================================
+%% 8. Create JSON sidecar
+if bSaveJson

```

Comment:
Related to my comments above. This might change. Cause bSaveJson shjould be about loading the reference, not about saving. If we don't load reference, but still provide fields, we should anyway save....

Reply:
Fixed in a768f82c2f482e9cdba3e7859302cf4b87dc2de3